src/tests/dfio/SConscript

```diff
@@ -0,0 +1,31 @@
+"""Build dfio plugin"""
+import daos_build
```

Comment:
(pylint-unused-import) Unused import daos_build

Reply:
You should actually build using daos_build to get proper rpath origin settings in which case this warning would go away.